invenio_app_rdm/config.py

```diff
@@ -689,6 +689,25 @@ def files_rest_permission_factory(obj, action):
     "requests": "/me/requests",
 }
 
+APP_RDM_USER_DASHBOARD_SUBMENUS = {
```

Comment:
I have some concerns about expanding the config variable matrix for small UI improvements, I would rather try to find more sustainable solution, but I need some time to propose something, please reach me back to discuss